### PR TITLE
Navbar refactor

### PR DESCRIPTION
**Pull request by vidoseaver**
Merge unmerged branches to avoid future merge conflicts
Updates mentor and student serializers.
Merge branch 'development' into fix-images-on-prod
unmerged branches merged
Move navbar to body of application.html.erb
about for images
fix spelling

### CODE REVIEW

**Comment by bfpepper** [comment]
looks good.

**Comment by vidoseaver** [comment]
i worked on this with ben. good to go

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<nav>
+ <div class="nav">
+  <a href="/" class="brand-logo">MENTOR<span class="ship-word-navbar">SHIP</span></a>
+   <ul id="nav-mobile" class="right hide-on-med-and-down">
+     <li><a href="/">Home</a></li>
+     <li><a href="/about">How It Works</a></li>
+     <li><a href='/auth/census'>Log In With Census</a></li>
+   </ul>
+ </div>
+</nav>
```
Comment (meganft) [inline app/views/layouts/_navbar.html.erb]: good work moving the navbar into erb!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -11,6 +11,20 @@
...
 </head>
 <body>
+  <nav>
+   <div class="nav">
+    <a href="/" class="brand-logo">MENTOR<span class="ship-word-navbar">SHIP</span></a>
+     <ul id="nav-mobile" class="right hide-on-med-and-down">
+       <li><a href="/">Home</a></li>
+       <li><a href="/about">How It Works</a></li>
+       <% if current_user %>
+        <li><a href='/logout'>Logout</a></li>
```
Comment (meganft) [inline app/views/layouts/application.html.erb]: 👍  to current_user

**Pull request merged**